gpu_offload.py

```diff
@@ -0,0 +1,12 @@
+# attempt to offload clip inference to GPU machine
```

Comment:
i dont think this belong to this pr (?)

Reply:
no, this does not...  only the cloud-config should be there

Reply:
I deleted the file.